test.js

```diff
@@ -170,7 +170,7 @@ test('removing regex pattern without other keys is supported', function (t) {
   t.plan(2)
 
   var instance = bloomrun()
-  var pattern = { to: /.*/ }
+  var pattern = { to: /^[a-zA-Z0-9-.]+$/i }
```

Comment:
Can you add a new test instead?

Reply:
only if you will suggest description of it

Reply:
name the one with the wildcard ' - wildcard case'.